test_apps/test_app/test/embarkJS_spec.js

```diff
@@ -0,0 +1,20 @@
+/*global describe, it, web3*/
+const assert = require('assert');
+const EmbarkJS = require('Embark/EmbarkJS');
+
+describe("EmbarkJS functions", function() {
+  it('should have access to ENS functions', async function() {
+    const rootAddress = await EmbarkJS.Names.resolve('embark.eth');
+    assert.strictEqual(rootAddress, web3.eth.defaultAccount);
+
+    const rootName = await EmbarkJS.Names.lookup(rootAddress);
+    assert.strictEqual(rootName, 'embark.eth');
+  });
+
+  it('should have access to Storage functions', async function() {
+    // Not calling the functions because it requires Swarm
+    assert.ok(EmbarkJS.Storage.saveText);
+    assert.ok(EmbarkJS.Storage.get);
+  });
+});
```

Comment:
I remember when trying this task, I was able to do that without much trouble too.
But, are you able to build a contract easily with EmbarkJS.Blockchain.

That was my main issue, I wasn't able to provide an easy way to make it available

Reply:
I'll try.

Reply:
It works :)
Will push a new test to show.

Reply:
Ohh nice